x-pack/plugins/triggers_actions_ui/public/application/lib/check_action_type_enabled.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCard, EuiLink } from '@elastic/eui';
+import { ActionType } from '../../types';
+import { VIEW_LICENSE_OPTIONS_LINK } from '../../common/constants';
+import './check_action_type_enabled.scss';
+
+export interface IsEnabledResult {
+  isEnabled: true;
+}
+export interface IsDisabledResult {
+  isEnabled: false;
+  message: string;
+  messageCard: JSX.Element;
+}
+
+export function checkActionTypeEnabled(
+  actionType?: ActionType
+): IsEnabledResult | IsDisabledResult {
+  if (actionType?.enabledInLicense === false) {
```

Comment:
Yeah, the scenario where it would be null is when we're waiting for the API to return the action type. I figured it's better to pretend the action type is enabled (more likely) until we know it's disabled and then display the messages in the UI. Thoughts?

Reply:
Was actually thinking the same thing, default to enabled, so 👍